config/core/configmaps/default-pingsource.yaml

```diff
@@ -15,13 +15,29 @@
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: config-ping-webhook
+  name: config-ping-defaults
   namespace: knative-eventing
   labels:
     eventing.knative.dev/release: devel
   annotations:
-    knative.dev/example-checksum: "6eaeecba"
+    knative.dev/example-checksum: "01b3f6e0"
 data:
-  ping-config: |
-    # dataMaxSize: 10  # Max number of bytes allowed to be sent for message excluding any base64 decoding.
-                       # Default is no limit set for data
+  _example: |
+    ################################
+    #                              #
+    #    EXAMPLE CONFIGURATION     #
+    #                              #
+    ################################
+
+    # This block is not actually functional configuration,
+    # but serves to illustrate the available configuration
+    # options and document them in a way that is accessible
+    # to users that `kubectl edit` this config map.
+    #
+    # These sample configuration options may be copied out of
+    # this example block and unindented to be in the data block
+    # to actually change the configuration.
+
+    # Max number of bytes allowed to be sent for message excluding any
+    # base64 decoding.  Default is no limit set for data
+    dataMaxSize: 4096
```

Comment:
Nit: the example value usually reflects the default.

Reply:
nit: If you are picky, remove the extra space after `decoding.`

Reply:
done